tests/CommentCtrlTest.php

```diff
@@ -17,6 +17,6 @@ public function testBlockComment(){
         $count = $comment->getNew($id,false);
 
 
-        PHPUnit::assertEquals(1,$count);
```

Comment:
The CommentCtrlTest chooses a comment for a block and saves the block id. 

The Method getNew returns the amount of comments for this block.

The problem is that a block could have multiple comments, so a check for only one was wrong !resolved